Cargo.toml

```diff
@@ -146,6 +146,7 @@ trustify-test-context = { path = "test-context" }
 # This will fail the build on targets where we need vendoring for openssl. Using rustls instead works around this issue.
 postgresql_archive = { version = "0.14.0", default-features = false, features = ["theseus", "rustls-tls"] }
 postgresql_embedded = { version = "0.14.0", default-features = false, features = ["theseus", "rustls-tls"] }
+postgresql_commands = { version = "0.14.0", default-features = false, features = ["tokio"] }
```

Comment:
why have we dropped rusttls-tls and theseus ?

Reply:
for `postgresql_commands`? Those features aren't available on this crate.